src/libs/es_writer.py

```diff
@@ -8,7 +8,7 @@
 # All the API logging is forwarded to the uWSGI server and gets written into the log file `uwsgo-entity-api.log`
 # Log rotation is handled via logrotate on the host system with a configuration file
 # Do NOT handle log file and rotation via the Python logging to avoid issues with multi-worker processes
-logging.basicConfig(format='[%(asctime)s] %(levelname)s in %(module)s:%(lineno)d: %(message)s', level=logging.DEBUG, datefmt='%Y-%m-%d %H:%M:%S')
```

Comment:
Same as above, need DEBUG level to print out the detailed error messages.

Reply:
Logging level reverted to DEBUG in branch file.